server/lib/tuist_web/marketing/controllers/marketing_html/home_next.html.heex

```diff
@@ -0,0 +1,553 @@
+          <img
+            src="/marketing/images/testimonials/trendyol/background.webp"
+            alt=""
+          />
+        </div>
+        <header data-part="content">
+          <div data-part="name">
+            <.trendyol_testimonial_logo />
+            <.button
+              href="#"
+              variant="secondary"
+              label="Read customer story"
+            >
+              <:icon_right><.chevron_right /></:icon_right>
+            </.button>
+          </div>
+          <h3>{gettext("Trendyol reduced build times by 65% with Tuist")}</h3>
+        </header>
+        <div data-part="graphic">
+          <img src="/marketing/images/testimonials/trendyol/graphic.webp" alt="Trendyol graph" />
+        </div>
+      </article>
+
+      <article data-part="customer-two">
+        <div data-part="background" aria-hidden="true">
+          <img
+            src="/marketing/images/testimonials/bumble/background.webp"
+            alt=""
+          />
+        </div>
+        <header data-part="content">
+          <div data-part="name">
+            <.bumble_testimonial_logo />
+            <.button
+              href="#"
+              variant="secondary"
+              label="Read customer story"
+            >
+              <:icon_right><.chevron_right /></:icon_right>
+            </.button>
+          </div>
+          <h3>{gettext("Bumble solved key issues without changing the core dev experience")}</h3>
+        </header>
+        <div data-part="graphic">
+          <img src="/marketing/images/testimonials/bumble/graphic.webp" alt="Bumble graphic" />
+        </div>
+      </article>
+    </section>
+
+    <section data-part="features-title">
+      <h2>{gettext("Powered by")}
+        <span data-part="box">{gettext("open source")}</span></h2>
+    </section>
+
+    <section data-part="stats" aria-label="Platform statistics">
+      <div data-part="column">
+        <article data-part="github-card">
+          <div data-part="background">
+            <img src="/marketing/images/stats/github.webp" alt="" />
+          </div>
+          <div data-part="content">
+            <h1>{gettext("5K+")}</h1>
+            <p>{gettext("Stars on GitHub")}</p>
+          </div>
+        </article>
+
+        <article data-part="oss-card">
+          <div data-part="background">
+            <img src="/marketing/images/stats/oss.webp" alt="" />
+          </div>
+          <div data-part="content">
+            <h1>{gettext("7+")}</h1>
+            <p>{gettext("Years of building OSS Foundation")}</p>
+          </div>
+        </article>
+      </div>
+
+      <div data-part="column">
+        <article data-part="minutes-card">
+          <div data-part="background">
+            <img src="/marketing/images/stats/minutes.webp" alt="" />
+          </div>
+          <div data-part="content">
+            <h1>{gettext("200,344,123")}</h1>
+            <p>{gettext("Millions of minutes saved")}</p>
+          </div>
```

Comment:
we don't have a data for this and I don't think it will be easy to get it. I'd omit this metric in the initial release of the home page